scala-package/core/src/main/scala/org/apache/mxnet/FILEHASH

```diff
@@ -0,0 +1,4 @@
+309G_O39xojNEt3DXVWNDQ
```

Comment:
Is this meant to be checked in? what files does this hash represent? how are they going to be updated?

Reply:
Good question. The hashfile contains the signature files that on Scala package not uploaded here. It follows the following format:
```Scala
abstract class SymbolAPIBase {
   /**
    * Comments
    */
   def Activation(...Args...) : Symbol
}
```
I used md5 hash in here:
```
  def MD5Generator(input : String) : String = {
    val md = MessageDigest.getInstance("MD5")
    md.update(input.getBytes("UTF-8"))
    val digest = md.digest()
    org.apache.commons.codec.binary.Base64.encodeBase64URLSafeString(digest)
  }
```
It basically take the Generated string into hash and store in this file. It should be the same across the different platform. The reason to do is is to avoid operator changes in Symbol and NDArray. One example, if one contributor changed the operators in NDArray and Symbol, he/she should comment the `require` line explained before in Scala package and update this file and also tag Scala Contributors to let them know there is a change in operator so we can determine if this will influence the user experience on Scala.